CHANGELOG.md

```diff
@@ -78,6 +78,7 @@
 * [ENHANCEMENT] `_config.job_names.<job>` values can now be arrays of regular expressions in addition to a single string. Strings are still supported and behave as before. #4543
 * [ENHANCEMENT] Queries dashboard: remove mention to store-gateway "streaming enabled" in panels because store-gateway only support streaming series since Mimir 2.7. #4569
 * [BUGFIX] Ruler dashboard: show data for reads from ingesters. #4543
+* [BUGFIX] Dod selector regex for deployments: change `(.*-mimir-)` to `(.*mimir-)`. #4603
```

Comment:
I'm not sure what "Dod" is supposed to be here. "Fix"?

Reply:
Typo, should have been `Pod`.